package.json

```diff
@@ -3,10 +3,14 @@
   "version": "1.0.0-beta.1",
   "description": "Discover, Install, Uninstall, Submit, Remove Adobe App Builder templates",
   "dependencies": {
+    "@adobe/aio-cli-lib-app-config": "^0.2.2",
+    "@adobe/aio-lib-console": "^3.2.0",
+    "@adobe/aio-lib-console-project-installation": "github:adobe/aio-lib-console-project-installation#DEVX-2298",
```

Comment:
`npm run test` is failing because it can't find this github repo for some reason.

Reply:
Could not reproduce. However, this will be updated once we have an NPM package to point it to.